tests/test_device.py

```diff
@@ -163,6 +164,19 @@ def get_device(wires=1):
         yield get_device
 
 
+with qml.tape.QuantumTape() as tp1:
+    qml.RY(0.1, wires=0)
+    qml.RX(0.2, wires=1)
+    qml.CNOT(wires=[0, 1])
+    [qml.expval(qml.PauliZ(0)), qml.expval(qml.PauliY(1))]
+
+with qml.tape.JacobianTape() as tp2:
+    qml.RY(0.1, wires=0)
+    qml.RX(0.2, wires=1)
+    qml.CNOT(wires=[0, 1])
+    qml.expval(qml.PauliZ(0))
```

Comment:
I suggest moving these either directly above the `TestBatchExecution` class, or even better, as class attributes. This keeps them 'closer' to the tests.

Reply:
I have to admit, I always forger how to use them as class attributes. If I move them into the class, then `tp1` is not defined in the tests. Do I need to do `self.tp1 = tp1` or something like that?

Reply:
If you have them as a class attribute, it just means using `self.tp1` inside the test functions themselves. Or you could have them above the test class, and just keep the tests as is